src/scss/index.scss

```diff
@@ -1 +1,109 @@
+@import "variables";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
 
+$enable-shadows: true;
```

Comment:
Lonely variable? Can you move this with the rest?

Reply:
I realized I actually can delete the lonely enable-shadow variable in the index.scss, as I have already enabled Bootstrap shadows in the _variables.scss, please correct me if I'm wrong.